src/host/_output.cpp

```diff
@@ -290,7 +290,7 @@ void WriteToScreen(SCREEN_INFORMATION& screenInfo, const Viewport& region)
         if (screenInfo.HasAccessibilityEventing())
         {
             auto endingCoordinate = startingCoordinate;
-            bufferSize.MoveInBounds(cellsModified, endingCoordinate);
+            bufferSize.MoveInBounds(gsl::narrow<til::CoordType>(cellsModified), endingCoordinate);

```

Comment:
<img width="222" alt="image" src="https://user-images.githubusercontent.com/189190/171739469-5d446348-958b-4ec6-8b61-9e25eb715fa6.png">

I'm surprised GSL lets you do a cast where `std::is_same<T,U>`

Reply:
What i mean is, `cellsModified` came from `GetInputDistance` and is therefore already the right type.. right?

Reply:
`cellsModified` is a `size_t` unfortunately (and an out parameter of the function).
I've changed the code to use the `CoordType` type.